outputs.tf

```diff
@@ -116,3 +116,24 @@ output "copy_to_glacier_lambda_arn" {
 #   description = "The ARN of the db parameter group"
 #   value       = "${module.db.this_db_parameter_group_arn}"
 # }
+
+
+output "get_message_queue_arn" {
```

Comment:
Make sure output names and descriptions are replicated across the `sqs/output.tf`, `orca/output.tf` and the main `output.tf` files.